server/src/db/schemas/compliance.js

```diff
@@ -0,0 +1,88 @@
+const mongoose = require('mongoose');
+
+// SCHEMA FORM CONTENT
+const ComplianceDetailSchema = new mongoose.Schema({
+    bbSpecification: {
+        type: String,
+        required: true
+    },
+    bbVersion: {
+        type: String,
+        required: true
+    },
+    status: {
+        type: String,
+        enum: ['in review', 'approved', 'rejected'],
+        default: 'in review'
+    },
+    submissionDate: {
+        type: Date,
+        default: Date.now
+    },
+    deploymentCompliance: {
+        isCompliant: {
+            type: Boolean,
+            required: true
+        },
+        details: {
+            type: String
+        }
+    },
+    requirementSpecificationCompliance: {
+        level: {
+            type: Number,
+            enum: [-1, 1, 2]
+        },
+        note: String
+    },
+    interfaceCompliance: {
+        level: {
+            type: Number,
+            enum: [-1, 1, 2]
+        },
+        note: String
+    }
+});
+  
+  const ComplianceVersionSchema = new mongoose.Schema({
+    version: {
+      type: String,
+      required: true
+    },
+    bbDetails: {
+      type: Map,
+      of: ComplianceDetailSchema,
+      required: true
+    }
+  });
+  
+  const ComplianceReportSchema = new mongoose.Schema({
+    softwareName: {
+      type: String,
+      required: true
+    },
+    logo: {
+      type: String,
+      required: true
+    },
+    website: {
+      type: String,
+      required: true,
+      match: /^http[s]?:\/\/.*/
```

Comment:
while this is good to have http prefix in regex, we need to be careful to either add validation on frontend or to add it it missing in use case implementaiton

Reply:
Okay I will keep the schema simple and delete this regex from 'website' and 'documentation'.
It will annotate this in other ticket to implement this